lib/superstore/inspect.rb

```diff
@@ -4,7 +4,7 @@ def inspect
       inspection = ["#{self.class.primary_key}: #{id.inspect}"]
 
       @attributes.each do |attribute, value|
-        if value.present?
+        if value.present? || value === false
```

Comment:
Tried to think of a single method to replace `present?`, but I don't think there is returns true for `false`, but still maintains the rest of the behavior. This seems like the best option.